Assets/Scripts/Scenes/InGame/Player/TempPlayerMove.cs

```diff
@@ -155,9 +156,9 @@ void Update()
             }
 
             //�������Ԃł���Γ���
-            if (_isCanMove)
+            if (_isCanMove && _isCannotMoveByParalyze == false)
                 Move();
-            else
+            else if(_isCanMove == false || _isCannotMoveByParalyze)

```

Comment:
```suggestion
 else if(!_isCanMove || _isCannotMoveByParalyze)
``` 
隣の_isCannotMoveByParalyzeが==trueを省略しているので_isCanMove == falseも省略して!_isCanMove とするのが読みやすいとお思います